MIT.md

```diff
@@ -0,0 +1,10 @@
+## Copyright 2023, [Shaun Kudzai, Binyam Olango]
```

Comment:
- [ OPTIONAL ] Hello. Since this is a solo project, consider updating your MIT license to reflect that state. What do you think? Perhaps `Binyam Olango` should not be included in the license.